_posts/2019-09-05-community-update.md

```diff
@@ -0,0 +1,146 @@
+
+## Flink Documentation 2.0
+
+Besides the translation effort, the community has also been working quite hard on a **Flink docs overhaul**. The main goals are to:
+
+ * Organize and clean-up the structure of the docs;
+ <p></p>
+ * Align the content with the overall direction of the project;
+ <p></p>
+ * Improve the _getting-started_ material and make the content more accessible to different levels of Flink experience. 
+
+Given that there has been some confusion in the past regarding unclear definition of core Flink concepts, one of the first completed efforts was to introduce a [Glossary](https://ci.apache.org/projects/flink/flink-docs-release-1.9/concepts/glossary.html#glossary) in the docs. To get up to speed with the roadmap for the remainder efforts, you can refer to [this FLIP](https://cwiki.apache.org/confluence/display/FLINK/FLIP-42%3A+Rework+Flink+Documentation) and the corresponding [umbrella Jira ticket](https://issues.apache.org/jira/browse/FLINK-12639).
+
+## Improvement of the Contribution Process and Experience
+
+The [guidelines](https://flink.apache.org/contributing/how-to-contribute.html) to contribute to Apache Flink have been reworked on the website, in an effort to lower the entry barrier for new contributors and reduce the overall friction in the contribution process. In addition, there is an ongoing vote to officialize a set of [bylaws](https://cwiki.apache.org/confluence/pages/viewpage.action?pageId=120731026) to help the community collaborate and coordinate more smoothly.
+
+For code contributors, a [Code Style and Quality Guide](https://flink.apache.org/contributing/code-style-and-quality-preamble.html) that captures the expected standards for contributions was also added to the "Contributing" section of the Flink website.
+
+It's important to stress that **contributions are not restricted to code**. Non-code contributions such as Mailing List support, documentation work or organization of community events are equally as important to the development of the project and highly encouraged.
+
+## New Committers and PMC Members
+
+The Apache Flink community has welcomed **5** new Committers and **3** PMC (Project Management Committee) Members in 2019, so far:
+
+### PMC
+	Thomas Weise, Jincheng Sun, Kete (Kurt) Young
+
+### Committers
+	Zhijiang Wang, Jiangjie (Becket) Qin, Rong Rong, Andrey Zagrebin, Hequn
+
+Congratulations and thank you for your hardworking commitment to Flink!
+
+# The Bigger Picture
+
+Flink continues to push the boundaries of <strike>stream</strike> data processing, and the community is proud to see an ever-increasingly diverse set of contributors, users and technologies join the ecosystem. 
+
+In the timeframe of three releases, the project jumped from **112 to 190 contributors**, also doubling down on the number of requested changes and improvements. To top it off, the Flink GitHub repository recently reached the milestone of **10k stars**, all the way up from the incubation days in 2014.
```

Comment:
This is the contributors per release?
https://github.com/apache/flink shows 554 contributors

Reply:
I didn't see the picture when reviewing the markdown :)